docs/clean-files.md

```diff
@@ -24,6 +24,7 @@ Clean files command is used to remove the Compacted, Marked For Delete ,In Progr
    ```
    CLEAN FILES FOR TABLE TABLE_NAME
    ```
+The above clean files command will clean Marked For Delete and Compacted segments depending on ```MAX_QUERY_EXECUTION_TIME``` (default 1 hr) and ``` carbon.trash.retention.days``` (default 7 days). It will also delete the timestamp subdirectories from the trash folder after expiration day(default 7 day, can be configured)
```

Comment:
```suggestion
The above clean files command will clean Marked For Delete and Compacted segments depending on ```max.query.execution.time``` (default 1 hr) and ``` carbon.trash.retention.days``` (default 7 days). It will also delete the timestamp subdirectories from the trash folder after expiration day(default 7 day, can be configured)
```

Reply:
done